cpp/velox/tests/ArrowToVeloxTest.cc

```diff
@@ -118,10 +118,10 @@ TEST_F(ArrowToVeloxTest, decimalV2A) {
   {
     // only RowVector can convert to RecordBatch
     auto row = makeRowVector({
-        makeShortDecimalFlatVector({1000265, -35610, 0}, DECIMAL(10, 3)),
+        makeShortDecimalFlatVector({1000265000, -35610000, 0}, DECIMAL(10, 3)),
```

Comment:
@jinchengchenghh
 The RowVector store the [UnscaledShortDecimal ](https://github.com/facebookincubator/velox/blob/94644c62f34daf966d32dadb5a0e059530eeaff9/velox/type/UnscaledShortDecimal.h#L27) value and we need change the value from 1000265, -35610, 0 to 1000265000, -35610000, 0.